x-pack/legacy/plugins/integrations_manager/server/lib/ingest_pipelines.ts

```diff
@@ -3,3 +3,24 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+export function rewriteIngestPipeline(
+  pipeline: string,
+  substitutions: Array<{
+    source: string;
+    target: string;
+    templateFunction: string;
+  }>
+): string {
+  substitutions.forEach(sub => {
+    const { source, target, templateFunction } = sub;
+    // This fakes the use of the golang text/template expression {{SomeTemplateFunction 'some-param'}}
```

Comment:
Maybe add a comment about the beats style as well?

Reply:
Good point.